docs/user-guide/systemrequirements-zosmf.md

```diff
@@ -79,8 +79,8 @@ If your implementation uses an external security manager other than RACF (for ex
 - [Configure z/OS Management Facility for ACF2](https://techdocs.broadcom.com/us/en/ca-mainframe-software/security/ca-acf2-for-z-os/16-0/installing/configure-z-os-management-facility-for-ca-acf2.html)
 :::
 
-## z/OSMF REST services for the Zowe CLI
-The Zowe CLI uses z/OSMF Representational State Transfer (REST) APIs to work with system resources and extract system data. Ensure that the following REST services are configured and available.
+## z/OSMF REST services for Zowe clients
```

Comment:
@adam-wolfe @zFernand0 Should all the references to "Zowe CLI" in [this table](https://docs.zowe.org/stable/user-guide/systemrequirements-zosmf/#zosmf-rest-services-for-the-zowe-cli) be changed to "Zowe clients"?

Reply:
I think that would be fair. The only potential point of confusion I see is if we consider the App Framework to be a Zowe client component. 

If it does not exist already, it might be good to define Zowe clients or Zowe client software: "Zowe client projects produce software that is intended to be executed on personal computers" (that's a terrible definition, will have to think about that) to differentiate between Zowe components that are installed on the mainframe and those that are not.

Reply:
The updated "Zowe client projects" definition looks clear to me.